tests/shopify_python/test_git_utils.py

```diff
@@ -248,3 +248,71 @@ def test_autopep_files(tmpdir):
     for fixed_file in files_to_autopep:
         with open(fixed_file) as file_to_check:
             assert file_to_check.readlines() == file_lines
+
+
+def test_linter(tmpdir):
+    # type: ('py.path.LocalPath') -> None
+    open(os.path.join(str(tmpdir), '__init__.py'), 'w')
+
+    file_lines = [
+        "def foo():\n",
+        "  return 1\n"
+    ]
+    file_path = os.path.join(str(tmpdir), 'file.py')
+    with open(file_path, 'w') as file_to_write:
```

Comment:
I think you can also perform `tmpdir.join('file.py').write('\n'.join(file_lines))` (skipping the `'\n'.join` if you don't split the text) instead of this.